lucille-core/src/main/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnector.java

```diff
@@ -72,6 +76,14 @@ public DatabaseConnector(Config config) {
       otherSQLs = new ArrayList<>();
       otherJoinFields = null;
     }
+    ignoreFields = new HashSet<>();
+    if (config.hasPath("ignoreFields")) {

```

Comment:
This is about ignoring specific _columns_ of the table before we convert them into document fields. So the name "ignoreColumns" would be more accurate.